client/src/store/new-request-module.ts

```diff
@@ -2594,7 +2595,7 @@ export class NewRequestModule extends VuexModule {
       // (do not show error to user)
       console.error('getQuickSearch() =', msg) // eslint-disable-line no-console
       this.mutateQuickSearch(false)
-      this.startAnalyzeName()
+      await this.startAnalyzeName()
```

Comment:
This (and the other ones below) possibly caused some components to display/hide (ie, flash) incorrectly when subsequent code executed before THIS completed.